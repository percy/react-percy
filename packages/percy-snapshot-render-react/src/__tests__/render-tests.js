/**
 * @jest-environment jsdom
 */

/* eslint-env browser */

import React from 'react';
import render from '../';

let el;

beforeEach(() => {
  document.body.innerHTML = '';

  el = document.createElement('div');
  document.body.appendChild(el);
});

it('renders the specified React markup into the DOM element', () => {
  const markup = <div>some markup</div>;

  render(markup, el);

  expect(el.innerHTML).toMatchSnapshot();
});
