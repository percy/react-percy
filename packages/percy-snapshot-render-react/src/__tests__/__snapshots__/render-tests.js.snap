// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the specified React markup into the DOM element 1`] = `"<div data-reactroot=\\"\\">some markup</div>"`;
